lir/registry.py

```diff
@@ -32,9 +35,11 @@ def _get_attribute_by_name(name: str) -> Any:
             else:
                 attr = __import__('.'.join(parts))
 
+            logging.debug(f'successfully imported: {name}. Found {attr}')
             return attr
 
         except (ModuleNotFoundError, AttributeError):
+            logging.debug(f'failed to import: {". ".join(parts[:class_name_index])}.')
```

Comment:
Regel 27 en 42 kunnen gecombineerd worden (deze regel verwacht je veel tegen te komen):

```suggestion
            LOG.debug(f'import failed: {". ".join(parts[:class_name_index])} from package {". ".join(parts[class_name_index:])}.')
```